torch/csrc/jit/codegen/cuda/iter_visitor.cpp

```diff
@@ -31,6 +32,93 @@ void remove_visited(
   }
 }
 
+// Return all dependencies of a node including members of the node.
+class RecursiveDependencies : public OptInDispatch {
+ public:
+  static std::vector<Statement*> next(Statement* stmt) {
+    RecursiveDependencies find_next(stmt);
+    return find_next.next_stmts_;
+  }
+
+ private:
+  RecursiveDependencies() {}
+
+  RecursiveDependencies(Statement* stmt) {
+    handle(stmt);
+  }
+
+  using OptInDispatch::handle;
+
+  void handle(Statement* stmt) {
+    if (stmt->isVal()) {
+      handle(stmt->as<Val>());
+    } else if (stmt->isExpr()) {
+      handle(stmt->as<Expr>());
+    } else {
+      TORCH_INTERNAL_ASSERT(
+          false, "IterVisitor could not detect type in next_dispatch.");
+    }
+  }
```

Comment:
Is this overload necessary? Build and C++ tests seem fine without it.

Reply:
Likely unnecessary, removing.